src/components/GridKeyboardNavigation/__stories__/useGridKeyboardNavigationContext.stories.mdx

```diff
@@ -0,0 +1,109 @@
+import { useRef } from "react";
+import { useGridKeyboardNavigationContext, GridKeyboardNavigationContext } from "../GridKeyboardNavigationContext";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { createStoryMetaSettings } from "../../../storybook/functions/create-component-story";
+import { DummyNavigableGrid, LayoutWithInnerKeyboardNavigation } from "./useGridKeyboardNavigationContext.stories";
+import { Flex } from "../..";
+
+export const metaSettings = createStoryMetaSettings({
+  component: useGridKeyboardNavigationContext
+});
+
+<Meta 
+  title="Keyboard/useGridKeyboardNavigationContext" 
+  component={ useGridKeyboardNavigationContext } 
+  argTypes={ metaSettings.argTypes }
+  decorators={ metaSettings.decorators } 
+  />
+
+<!--- Component documentation -->
+
+# useGridKeyboardNavigationContext
+
+- [Overview](#overview)
+- [Usage](#usage)
+- [Arguments](#arguments)
+- [Returns](#returns)
+- [Feedback](#feedback)
+
+## Overview
+
+A hook used to specify a connection between multiple navigable components, which are navigable between each other.
+
+<Canvas>
+  <Story name="Overview">{
+    () => {
+      const wrapperRef = useRef(null);
+      const leftElRef = useRef(null);
+      const rightElRef = useRef(null);
+      const keyboardContext = useGridKeyboardNavigationContext([{leftElement: leftElRef, rightElement: rightElRef}], wrapperRef)
+      return (
+        <GridKeyboardNavigationContext.Provider value={keyboardContext}>
+          <Flex ref={wrapperRef}>
+            <DummyNavigableGrid ref={leftElRef} itemsCount={15} numberOfItemsInLine={4} itemPrefix="L "/>
```

Comment:
The story doesn't look too good, but I wanted to emphasize the layout structure so the behavior will be clear. (you can check the gif in the PR's description).
Do you think I should change it?